### ISSUE TITLE
Syncfusion Flutter PDF Viewer (26.1.42) Fails to Load PDF Generated by Syncfusion Flutter PDF

### ISSUE THREAD

**Issue by loveoverflowdev** [syncfusion/flutter-widgets#1988]
### Bug description

The Syncfusion Flutter PDF Viewer (26.1.42) fails to properly load and display a PDF file generated by the Syncfusion Flutter PDF package. 

The generated PDF file, containing two pages, successfully opens in other PDF viewer packages, but when attempting to view it with the Syncfusion PDF Viewer, the screen keeps loading indefinitely, displaying one page per two without throwing any exceptions.

### Steps to reproduce

pdfviewer_bug_reproduction zip: [pdfviewer_bug_reproduction.zip](https://github.com/user-attachments/files/16318918/pdfviewer_bug_reproduction.zip)

### 1. Create Project:

```
flutter create pdfviewer_bug_reproduction
cd pdfviewer_bug_reproduction
dart pub add syncfusion_flutter_pdfviewer:'^26.1.42'
```

### 2. Import Assets:
- PDF File: 
[sample.pdf](https://github.com/user-attachments/files/16318905/sample.pdf)
- Path: pdfviewer_bug_reproduction/sample.pdf
- YAML: 
```
assets:
    - assets/sample.pdf
```
- Additional Images:
![image](https://github.com/user-attachments/assets/2c25f0fd-84a3-4a4e-af8d-dbc9ca95d7e8)

### 3. Replace main.dart

```dart
// main.dart
import 'package:flutter/material.dart';
import 'package:syncfusion_flutter_pdfviewer/pdfviewer.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'PdfViewer Bug Reproduction',
      theme: ThemeData(
        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
        useMaterial3: true,
      ),
      home: const MyHomePage(title: 'PdfViewer Bug Reproduction'),
    );
  }
}

class MyHomePage extends StatelessWidget {
  final String title;

  const MyHomePage({
    required this.title,
    super.key,
  });

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text(title),
      ),
      body: SfPdfViewer.asset('assets/sample.pdf'),
    );
  }
}
```
### 4. Run
```
flutter run
```

## Results:
![image](https://github.com/user-attachments/assets/80c1d2f0-d41b-49bc-b431-17e267dfc39c)
The PDF Viewer screen keeps loading indefinitely and displays one page per two without any exceptions being thrown.

### Code sample

```dart
// main.dart
import 'package:flutter/material.dart';
import 'package:syncfusion_flutter_pdfviewer/pdfviewer.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  // This widget is the root of your application.
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'PdfViewer Bug Reproduction',
      theme: ThemeData(
        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
        useMaterial3: true,
      ),
      home: const MyHomePage(title: 'PdfViewer Bug Reproduction'),
    );
  }
}

class MyHomePage extends StatelessWidget {
  final String title;

  const MyHomePage({
    required this.title,
    super.key,
  });

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text(title),
      ),
      body: SfPdfViewer.asset('assets/sample.pdf'),
    );
  }
}
```

### Screenshots or Video

![image](https://github.com/user-attachments/assets/3ba883eb-e2a0-4faf-bd72-b276bd7ae203)

### Stack Traces

The PDF Viewer screen keeps loading indefinitely and displays one page per two without any exceptions being thrown.

### On which target platforms have you observed this bug?

Android

### Flutter Doctor output

```
flutter doctor 
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.22.3, on Microsoft Windows [Version 10.0.22631.3880], locale en-US)
[✓] Windows Version (Installed version of Windows is version 10 or higher)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Chrome - develop for the web
[✓] Visual Studio - develop Windows apps (Visual Studio Community 2022 17.9.6)
[✓] Android Studio (version 2023.3)
[✓] IntelliJ IDEA Community Edition (version 2024.1)
[✓] Connected device (4 available)
[✓] Network resources

• No issues found!
```

**Comment by loveoverflowdev** [comment]
This issue seem to be disappear when I downgrade version to syncfusion_flutter_pdfviewer: ^23.1.36
 instead of 26.1.42

**Comment by immankumarsync** [comment]
@loveoverflowdev, we are able to replicate the reported issue. Currently, we are validating this issue and we will update further details once the issue validation is completed.

**Comment by Daniel-Cong** [comment]
I experienced the same issue on version 26.1.42. Even when I am opening very light two pages pdf file, most of time, only one page was loaded, while the other one loads indefinitely.
The problem seems to be gone when I updated to version 26.2.4.

**Comment by immankumarsync** [comment]
@loveoverflowdev, we have confirmed that the "second page is not rendering in a two-page document" is a defect in our product and have logged a report for it. The fix for the same will be included in our upcoming weekly release.
Please follow the feedback link to track the status of the bug, https://www.syncfusion.com/feedback/59846/android-second-page-is-not-rendering-in-a-two-page-document

**Comment by loveoverflowdev** [comment]
I did see that this bug was fixed in ```syncfusion_flutter_pdf: ^26.2.7```.
So this issue should be closed
